components/markdown_image_expand/markdown_image_expand.scss

```
@@ -0,0 +1,95 @@
+@import "compass/css3";
+@import "sass/utils/functions";
+@import "utils/variables";
+
+.markdown-image-expand {
+    position: relative;
+
+    display: flex;
+    align-items: center;
+}
+
+.markdown-image-expand__collapse-button {
+    position: absolute;
+    top: 0;
+    left: 0;
+
+    display: none;
+    width: 24px;
+    height: 24px;
+    border: 1px solid var(--center-channel-color-16);
```

For these `-##` variables that include an opacity, we're trying to get away from using those since it adds a lot of extra CSS variables. Could you instead switch this to use `rgba` to specify that opacity instead? Like this one would look like
```suggestion
    border: 1px solid rgba(var(--center-channel-color-rgb), 0.16);
```

---

Fixed.